src/screens/users/index.js

```diff
@@ -18,49 +26,87 @@ const Users = ({ users, onSearch, onChatCreate }) => {
     onSearch(name);
   };
 
-  const handleCreateChat = (userId) => {
+  const handleCreateChat = userId => () => {
     onChatCreate(userId);
   };
 
+  const handleAddUser = (...args) => () => {
+    onAddUserToChat(...args);
+  };
+
   return (
     <View style={styles.container}>
-      <TextInput
-        onChangeText={handleChange('name')}
-        autoCapitalize="none"
-        style={styles.input}
-        placeholder="Name"
-        placeholderTextColor={COLORS.BLUE}
-      />
-      <Touchable
-        onPress={handleSubmit}
-        style={styles.button}
-      >
-        <Text style={styles.buttonText}>Search</Text>
-      </Touchable>
-      <ScrollView style={styles.usersContainer}>
-        {
-          users.map(({ _id, name }) => (
-            <Touchable
-              key={_id}
-              style={styles.userRow}
-              onPress={() => handleCreateChat(_id)}
-            >
-              <Text style={styles.userName}>{name}</Text>
-            </Touchable>
-          ))
-        }
-      </ScrollView>
+      {
+        pending
+          ? <ActivityIndicator size="large" color="#0000ff" />
+          : (
+            <Fragment>
+              <TextInput
+                onChangeText={handleChange('name')}
+                autoCapitalize="none"
+                style={styles.input}
+                placeholder="Name"
+                placeholderTextColor={COLORS.BLUE}
+              />
+              <Touchable
+                onPress={handleSubmit}
+                style={styles.button}
+              >
+                <Text style={styles.buttonText}>Search</Text>
+              </Touchable>
+              <ScrollView style={styles.usersContainer}>
+                {
+                  users.map(({ _id, name, isOnline }) => (
+                    <View
+                      style={styles.userRow}
+                      key={_id}
+                    >
+                      <View style={styles.infoSection}>
+                        <Text style={styles.userName}>{name}</Text>
+                        <Text style={[
+                          styles.online,
+                          isOnline
+                            ? { color: COLORS.GRAY }
+                            : { color: COLORS.GRAY_LIGHTER }]}
+                        >
+                          {isOnline ? 'Online' : 'Offline'}
+                        </Text>
+                      </View>
+                      <View style={styles.controlsSection}>
+                        <Touchable
+                          onPress={
+                            Object.keys(params).length > 0 && params.addindUser
```

Comment:
use
`Object.keys(params).length`
instead 
`Object.keys(params).length > 0`
case `0` is **Falsy** value

**line 84 the same**